phpunit.xml.dist

```diff
@@ -6,7 +6,11 @@
          executionOrder="random"
          failOnWarning="true"
          failOnRisky="true"
+         failOnDeprecation="true"
```

Comment:
I think that's a good idea to fail builds on deprecation so we won't introduce more deprecation